### PR TITLE
Hooks refactor

### PR DESCRIPTION
**Pull request by aditudorache**
This PR does a refactoring from all the old style react components to hooks along with updating all the packages to the last version.
There is yet a known issue: the rotation inside an svg item is not correct done. This is due the optimalizations in the react-scripts, will solve that when we decide if we need to eject the configuration.

### CODE REVIEW

File: src/shared/hooks/useDataFeatching.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+function useDataFetching() {
+  const [results, setResults] = React.useState(null);
+  const [errorMessage, setErrorMessage] = React.useState(false);
+  const [loading, setLoading] = React.useState(false);
+
+  async function fetchData(endpoint) {
+    setLoading(true);
+    try {
+      const data = await getByUrl(endpoint);
+      console.log(data);
```
Comment (janjaap) [inline src/shared/hooks/useDataFeatching.js]: Forgot to remove

File: src/shared/hooks/useDataFeatching.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      setResults(data);
+    } catch (e) {
+      setErrorMessage(e.message);
+    }
+
+    setLoading(false);
+    // console.log(results);
```
Comment (janjaap) [inline src/shared/hooks/useDataFeatching.js]: And here

File: src/shared/hooks/useDataFeatching.js (hunk 1 above)
Comment (aditudorache) [reply src/shared/hooks/useDataFeatching.js]: thanks

File: src/shared/hooks/useDataFeatching.js (hunk 2 above)
Comment (aditudorache) [reply src/shared/hooks/useDataFeatching.js]: thanks

File: src/components/App.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+import styles from './App.module.scss';
+import Header from './header/Header';
+import DashboardPage from 'views/dashboard/DashboardPage';
+import ConceptPage from 'views/concepts/ConceptPage';
+import ContactPage from 'views/contact/ContactPage';
+import { appRoutes } from 'constants.js';
+import LandingPage from '../views/landing/LandingPage';
+import { trackPageView } from '../helpers';
+import withKeycloak from '../hoc/withKeyCloak';
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply src/components/App.jsx]: fair point, I moved this to a custom hook

**Pull request merged**